x-pack/plugins/serverless_search/public/application/components/overview.tsx

```diff
@@ -86,11 +86,15 @@ export const ElasticsearchOverview = () => {
       </EuiPageTemplate.Section>
       <EuiPageTemplate.Section color="subdued" bottomBorder="extended">
         <SelectClientPanel>
-          <LanguageClientPanels
-            setSelectedLanguage={setSelectedLanguage}
-            languages={languageDefinitions}
-            selectedLanguage={selectedLanguage}
-          />
+          {languageDefinitions.map((language, index) => (
+            <EuiFlexItem key={`panelItem.${index}`}>
+              <LanguageClientPanel
```

Comment:
Instead of creating the languages panel map in `select_client`, moved to `overview.tsx.` It helps with not passing the props to `select_client ` (_child_) which don't really use them but only used by `language_client_panel `(_grandchild_)